influxql/task_manager.go

```diff
@@ -95,15 +95,15 @@ func (t *TaskManager) executeShowQueriesStatement(q *ShowQueriesStatement) (mode
 	for id, qi := range t.queries {
 		d := now.Sub(qi.startTime)
 
-		var ds string
-		if d == 0 {
-			ds = "0s"
-		} else if d < time.Second {
-			ds = fmt.Sprintf("%du", d)
-		} else {
-			ds = (d - (d % time.Second)).String()
+		if d >= time.Second {
```

Comment:
Updated to use `switch` instead. I'll merge when it hits green unless something else is found. Thanks.